src/test/java/io/skai/okta/internshipstreamingserviceparserxml/repository/EpisodeAdditionalInfoRepositoryTest.java

```diff
@@ -0,0 +1,60 @@
+package io.skai.okta.internshipstreamingserviceparserxml.repository;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.EpisodeAdditionalInfo;
+import io.skai.okta.internshipstreamingserviceparserxml.repository.impl.EpisodeAdditionalInfoRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+public class EpisodeAdditionalInfoRepositoryTest {
+
+    @Autowired
+    private EpisodeAdditionalInfoRepository repository;
+
+    private EpisodeAdditionalInfo expectedInfo;
+
+    @BeforeEach
+    void setUp() {
+        expectedInfo = EpisodeAdditionalInfo.builder()
+                                            .parentId(9379992)
+                                            .imdbID("test imdb id")
+                                            .seriesID("test series id")
+                                            .imdbRating("test imdb rating")
+                                            .imdbVotes("test imdb votes")
+                                            .runtime("test run time")
+                                            .build();
+    }
+
+    @Test
+    public void testRepositoryShouldBeNotNull() {
+        assertNotNull(repository);
```

Comment:
what this test verifies?

Reply:
have an issue before. repository was null.
this is also redundant, because if i do testEpisodeSuccessfulSaveAndGet i will have NPException there? i should remove this assertion or move it in testEpisodeSuccessfulSaveAndGet method?